node/src/types/json_compatibility/execution_result.rs

```diff
@@ -137,6 +219,47 @@ impl From<&Op> for Operation {
     }
 }
 
+impl ToBytes for Operation {
+    fn to_bytes(&self) -> Result<Vec<u8>, bytesrepr::Error> {
+        let mut buffer = bytesrepr::allocate_buffer(self)?;
+        match self {
+            Operation::Read => {
+                buffer.insert(0, OP_READ_TAG);
+            }
+            Operation::Write => {
+                buffer.insert(0, OP_WRITE_TAG);
+            }
+            Operation::Add => {
+                buffer.insert(0, OP_ADD_TAG);
+            }
+            Operation::NoOp => {
+                buffer.insert(0, OP_NOOP_TAG);
+            }
+        }
+        Ok(buffer)
+    }
+
+    fn serialized_length(&self) -> usize {
+        bytesrepr::U8_SERIALIZED_LENGTH
```

Comment:
`U8_SERIALIZED_LENGTH` should be imported.

Reply:
Imported accordingly